framework/src/kernels/ConservativeAdvection.C

```diff
@@ -21,7 +21,13 @@ ConservativeAdvectionTempl<is_ad>::generalParams()
 {
   InputParameters params = GenericKernel<is_ad>::validParams();
   params.addClassDescription("Conservative form of $\\nabla \\cdot \\vec{v} u$ which in its weak "
-                             "form is given by: $(-\\nabla \\psi_i, \\vec{v} u)$.");
+                             "form is given by: $(-\\nabla \\psi_i, \\vec{v} u)$. Velocity can be "
+                             "given as 1) a variable, for which the gradient is automatically "
+                             "taken, 2) a vector variable, or a 3) vector material.");
+  params.addParam<MaterialPropertyName>(
+      "velocity_scalar_coef",
```

Comment:
Ugh, this is a poor name (my bad)...it's not a scalar, it's a material property. Any opinions on changing this @GiudGiud or @lindsayad ?